app/components/contact-item/sortable-table.hbs

```diff
@@ -0,0 +1,103 @@
+<div class="auk-scroll-wrapper">
```

Comment:
Kan deze component niet gemerged worden met `contact-item/table.hbs`? Het enige verschil lijkt mij de sortable table headers. Je kan bepalen adhv de `@onSortChange` action handler of de table sortable moet zijn: als het argument meegegeven wordt wel; anders niet.